lib/pure/collections/tables.nim

```diff
@@ -1313,6 +1314,8 @@ proc initOrderedTable*[A, B](initialSize = defaultInitialSize): OrderedTable[A,
       a = initOrderedTable[int, string]()
       b = initOrderedTable[char, seq[int]]()
   initImpl(result, initialSize)
+  result.first = -1
+  result.last = -1
```

Comment:
This could be noise or a very relevant bugfix. Unfortunately I cannot know because it's a huge PR.

Reply:
These two lines I moved out of ``initImpl``. In there they where behind a ``when compiles`` so that they would be generated exclusively here. It is simpler to just put it here.

Reply:
Ok.